### DISCUSSION TITLE
Daily 21/9

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#19] [General]
Ayer realicé la instalación de Wordpress y me saqué varias dudas sobre cómo instalarlo que espero servirán cuando tengamos que hacerlo para el proyecto full-stack. Veo que varios subieron cambios a la wiki del ERS así que hoy paso los cambios al pdf para actualizarlo. También estoy buscando nombre para la aplicación, necesitamos varias sugerencias para tomar una decisión. Y no pude ingresar las tareas porque tuvimos reunión presencial en el trabajo así que voy a hacerlo hoy. Me gustaría repasar las lecciones de inglés antes de dar el examen, tal vez pueda hacer todo hoy pero por el repaso tal vez algo me quede para mañana.

**Comment by NataliaAlvarezIspc** [comment]
Ayer comence con el estudio de CSS, llegue hasta el capítulo de Selectores. 
Por otro lado estuve con la instalación de Wordpress que, como había previsto, me trajo complicaciones al momento de ejecurlo. Lo cual me demoró en el avance del estudio. Finalmente tuve que desinstalarlo e instalarlo nuevamente.
Hoy estare haciendo práctica de lo estudiado ayer en CSS y redactando los pasos que seguí para instalar Worpress, para subir la tarea de Sistema de Gestión de Contenidos. También haciendo un repaso de los temas de Inglés para el exámen. Y pensando historias de usuarios.
La dificultad que veo es que puede que el tiempo del que dispongo no me alcance para hacer todo.

**Comment by AhumadaCeleste** [comment]
Ayer estuve abocada a continuar con Html. Hoy prosigo con ello, pero más abocada a estudiar ingles. Mañana daré el examen de ingles y espero poder terminar con HTML e iniciar con css.

**Comment by Brian-Almada** [comment]
Ayer estuve intentando descargar Wordpress en local pero todo resultó sin  éxito. Hoy voy a intentar solucionar los problemas que tuve al descargarlo. Luego voy a continuar con Python.

**Comment by IngridAlcocer05** [comment]
Ayer instale XAMPP. Hoy seguí con la lectura de Python del ultimo modulo para poder rendir el examen y también intente instalar Wordpress pero se me dificulto.

**Comment by rocappellano** [comment]
Seguí con HTML, y estudiando inglés

**Comment by Thiagox749** [comment]
Hoy realicé el examen de ingles, repasé HTML y CSS. Mañana repasaré python y examinare posibles US para el proyecto.

**Comment by claubott** [comment]
Rendí inglés e instalé un cms para gestión de contenidos. Sin problemas. Luego subo la tarea en el foro del ispc.
